pygmt/tests/test_plot.py

```diff
@@ -495,3 +496,46 @@ def test_plot_deprecate_columns_to_incols(region):
         )
         assert len(record) == 1  # check that only one warning was raised
     return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_plot_ogrgmt_file_multipoint_default_style():
+    """
+    Make sure that OGR/GMT file with MultiPoint geometry are plotted as squares
```

Comment:
```suggestion
    Make sure that OGR/GMT files with MultiPoint geometry are plotted as squares
```
Either file**s** or ... geometry **is** plotted ...